src/tests/dispatch-payload.test.ts

```diff
@@ -12,33 +12,32 @@ import {
   generateViewSubmissionPayload,
 } from "./test_utils.ts";
 
+const noop = () => "";
+
 Deno.test("DispatchPayload function", async (t) => {
   await t.step("should be defined", () => {
     assertExists(DispatchPayload);
   });
-  await t.step(
-    "should throw if an unrecognized event type is dispatched",
-    async () => {
-      await assertRejects(() =>
-        DispatchPayload({
-          body: { type: "messinwitcha" },
-          context: { bot_access_token: "12345", team_id: "123", variables: {} },
-        }, () => [])
-      );
-    },
-  );
   await t.step(
     "should throw if no function callback_id present in payload",
     async () => {
-      await assertRejects(() =>
-        DispatchPayload({
-          body: { type: "function_executed", event: {} },
-          context: { bot_access_token: "12345", team_id: "123", variables: {} },
-        }, () => [])
+      await assertRejects(
+        () =>
+          DispatchPayload({
+            body: { type: "function_executed", event: {} },
+            context: {
+              bot_access_token: "12345",
+              team_id: "123",
+              variables: {},
+            },
+          }, noop),
+        Error,
+        "not find the function callback_id",
```

Comment:
In general I've found we should be asserting on the error message when checking for exceptions, as we throw exceptions with different messages in different circumstances and some of them can get confused. This can lead to the wrong exception being thrown, and us not realizing this because the test passes anyways.